runtime/calamari/src/weights/manta_collator_selection.rs

```diff
@@ -46,187 +46,207 @@ pub trait WeightInfo {
 	fn set_invulnerables(b: u32, ) -> Weight;
 	fn set_desired_candidates() -> Weight;
 	fn set_candidacy_bond() -> Weight;
+	fn set_eviction_baseline() -> Weight;
+	fn set_eviction_tolerance() -> Weight;
 	fn register_as_candidate(c: u32, ) -> Weight;
 	fn leave_intent(c: u32, ) -> Weight;
 	fn remove_collator(c: u32, ) -> Weight;
 	fn register_candidate(c: u32, ) -> Weight;
 	fn note_author() -> Weight;
-	fn new_session(r: u32, c: u32, ) -> Weight;
+	fn new_session(c: u32, ) -> Weight;
 }
 
 /// Weights for manta_collator_selection using the Substrate node and recommended hardware.
 pub struct SubstrateWeight<T>(PhantomData<T>);
 impl<T: frame_system::Config> manta_collator_selection::WeightInfo for SubstrateWeight<T> {
 	// Storage: CollatorSelection Invulnerables (r:0 w:1)
 	fn set_invulnerables(b: u32, ) -> Weight {
-		(10_564_000 as Weight)
-			// Standard Error: 15_000
-			.saturating_add((118_000 as Weight).saturating_mul(b as Weight))
+		(10_476_000 as Weight)
+			// Standard Error: 18_000
```

Comment:
How did you generate this weights file ?

Reply:
Originally, from the benchmark calamari github action right before #358 was merged. Back then I only applied the output file to Dolphin. I now copied it back from dolphin to Calamari.

Reply:
Can you just run the Calamari benchmark workflow on this branch

Reply:
Sure, I thought it won't be necessary since we'll rebenchmark on release building anyway, but i think it makes sense to see the exact impact of this changeset alone

Reply:
Updated. In short: Extrinsics lose weight, note_author (once per block) and new_session(once per session) get heavier